lib/http.js

```diff
@@ -304,6 +308,53 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     })
   );
 
+  // GET /ledger-agent/{AGENT_ID}/peers
+  // Get recommended peers
+  app.options(routes.peers, cors());
+  app.get(
+    routes.peers,
+    // FIXME: optionallyAuthenticated not working as expected
+    /*optionallyAuthenticated,*/ cors(), asyncHandler(async (req, res) => {
+      const {actor} = req.user || {};
+      const agentId = 'urn:uuid:' + req.params.agentId;
+      const options = {};
+
+      if(!req.user) {
+        options.public = true;
+      }
+
+      const ledgerAgent = await brLedgerAgentGet(
+        (req.user) ? actor : null, agentId, options);
+      const {ledgerNode} = ledgerAgent;
+
+      const result = await ledgerNode.peers.getRecommended();
+
+      res.json(result);
+    }));
+
+  // GET /ledger-agent/{AGENT_ID}/peers/{PEER_ID}
+  // Get a peer by ID
+  app.get(
+    routes.peer,
+    // FIXME: optionallyAuthenticated not working as expected
```

Comment:
@mattcollier what is the issue here?

Reply:
Well, at some point early on when I added the new route and test,  I was getting some error from deep within the bowels of bedrock-passport.  To avoid distraction, I just commented it out.  I've just uncommented the `optionallyAuthenticated` middleware, and everything is working properly.  So, that's been fixed and the fixmes have been removed.